decidim-elections/app/assets/javascripts/decidim/elections/trustee_zone/tally.js.es6

```diff
@@ -30,13 +33,25 @@ $(() => {
   );
   let currentStep = null;
 
+  // Use the correct trustee wrapper adapter
+  let trusteeWrapperAdapter = null;
+
+  if (schemeName === "dummy") {
```

Comment:
I take it this will be fixed in the future?

Reply:
Confirmed that this is done on purpose, and will be improved in the future.

Reply:
This PR uses the `DummyTrusteeWrapperAdapter` , the next PR by @beagleknight implements the `election_guard` scheme.